GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -194,6 +194,7 @@ T   Translate
 
         private readonly TranslationString _buildReportTabCaption = new("Build Report");
         private readonly TranslationString _consoleTabCaption = new("Console");
+        private readonly TranslationString _historyTabCaption = new("History");
```

Comment:
I don't like the "History" name of the tab. I found it too misleading because too close to the notion of "git history". I think the (new) users will expect to find something else displayed in a tab named like that.
I prefer something more like "Trace", "Output" or "Processes"...

Reply:
I think "Output" as tab label is well-known and fits best. I am going to use "OutputHistory" in class names.